<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,0,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_coming">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="AutoFontLabel" name="label_cooking">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#2a85c7;</string>
          </property>
          <property name="text">
           <string>准备中(Coming)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView_cooking"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="AutoFontLabel" name="label_9">
         <property name="minimumSize">
          <size>
           <width>5</width>
           <height>7</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>5</width>
           <height>7</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="AutoFontLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>5</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>5</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:0px;
border-right:5px dashed white;
border-left:5px dashed white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="AutoFontLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>5</width>
           <height>7</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>5</width>
           <height>7</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame_served">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="AutoFontLabel" name="label_service">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#4f5f85;</string>
          </property>
          <property name="text">
           <string>请取餐(Served)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView_service"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="AutoFontLabel" name="label_information">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>请留意您的取单号码取餐(Please check your number)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#frame{background-color:rgb(37, 48, 65)}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
      <item>
       <widget class="AutoFontLabel" name="label_soft_name">
        <property name="text">
         <string>KDS System</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_logo">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/fastfdbacklogo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="AutoFontLabel" name="label_company_name">
        <property name="text">
         <string>好搭档智能厨房显示系统</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AutoFontLabel</class>
   <extends>QLabel</extends>
   <header>autofontlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
